src/db/database.js

```diff
@@ -151,20 +178,19 @@ export default class Database {
         // items, you may introduce filter functions in the loop... see the template within comments.
         return collector;
     }
-    
-    __allBeveragesMoreDetailed(id, namn, pris, varugrupp, land, producent, alkoholhalt, forpackning) {
+
+    __allBeveragesMoreDetailed(id,namn,pris,varugrupp,land,producent,alkoholhalt,forpackning) {
         let collectorJSON = {
-                            "id": id,
-                            "namn": namn, 
-                            "pris": pris, 
-                            "varugrupp": varugrupp, 
-                            "land": land, 
-                            "producent": producent,
-                            "alkoholhalt": alkoholhalt,
-                            "forpackning": forpackning
-                            };
+            id: id,
+            namn: namn,
+            pris: pris,
+            varugrupp: varugrupp,
+            land: land,
+            producent: producent,
+            alkoholhalt: alkoholhalt,
+            forpackning: forpackning,
+        };
```

Comment:
Why is this changed? JSON objects are created in name/value-pairs, and they are surrounded by quotation marks.